lib/license_finder/packages/erlangmk_package.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'rubygems'
+
+class InvalidErlangmkPackageError < ArgumentError
+end
+
+# NOTE:
```

Comment:
@lukebakken Did you mean to leave these comments here?

Reply:
Sure did, so in a year if I have to fix something I know how to generate the deps output using `erlang.mk`.

Reply:
Feel free to remove them if you'd like. Otherwise I think this PR is set.

Reply:
I opened #768 with a question. If that's not the preferred way to ask something, let me know. I figure someone else may learn from it in the future.